src/kernels/hip_float8.hpp

```diff
@@ -555,7 +544,44 @@ class numeric_limits<miopen_f8::hip_f8<miopen_f8::hip_f8_type::bf8>>
     }
 };
 
+} // namespace miopen_f8
+
+#ifndef __HIPCC_RTC__
+namespace std {
+inline bool isfinite(miopen_f8::hip_f8<miopen_f8::hip_f8_type::fp8> x) // NOLINT
+{
+    return x.is_inf();
+}
```

Comment:
This should be:
```C++
return !(x.is_nan() || x.is_inf());
```
isn't it?

Reply:
Yes i copy pasted previous logic without realizing this.  This should be fixed